{"name": "test_vwo_login_positive", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'email'", "trace": "self = <tests.vwo_login_tests.test_vwo_login_pom.TestLogin object at 0x0000025EED8CD0D0>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"92387147cd6b18a32c8130cf06bc3ed9\")>\n\n    @allure.epic(\"VWO Login Test\")\n    @allure.feature(\"TC#1 - VWO App Positive Test\")\n    @pytest.mark.positive\n    @pytest.mark.usefixtures(\"setup\")\n    def test_vwo_login_positive(self, setup):\n        driver = setup\n        driver.get(self.base_url)\n        login_page = LoginPage(driver)\n>       login_page.login_to_vwo(email=self.email, password=self.password)\nE       AttributeError: 'TestLogin' object has no attribute 'email'\n\ntest_vwo_login_pom.py:35: AttributeError"}, "start": 1716207319962, "stop": 1716207328781, "uuid": "6a07acf9-6839-48de-9a08-c4bb99288aa6", "historyId": "a1afd2d2df25f43dc2a12081a263ef07", "testCaseId": "a1afd2d2df25f43dc2a12081a263ef07", "fullName": "tests.vwo_login_tests.test_vwo_login_pom.TestLogin#test_vwo_login_positive", "labels": [{"name": "epic", "value": "VWO Login Test"}, {"name": "feature", "value": "TC#1 - VWO App Positive Test"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "tag", "value": "positive"}, {"name": "parentSuite", "value": "tests.vwo_login_tests"}, {"name": "suite", "value": "test_vwo_login_pom"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Mani"}, {"name": "thread", "value": "20068-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwo_login_tests.test_vwo_login_pom"}]}